schema/store_item.go

```diff
@@ -12,5 +12,6 @@ type StoreItem struct {
 	RepoURL                string            `json:"repo_url"`
 	Environment            map[string]string `json:"environment"`
 	Labels                 map[string]string `json:"labels"`
+	Annotations            map[string]string `json:"annotations"`
```

Comment:
Always add new code last when changing structs, it can affect serialization.

Reply:
Ok I will know from now on.